src/context-parser-handlebars.js

```diff
@@ -404,6 +430,49 @@ ContextParserHandlebars.prototype.analyzeAst = function(ast, contextParser, char
 
                 output += t.output;
 
+            // if the 'partial expression' is not in Data State,
+            // TODO: we support basic partial only, need to enhance it.
+            // http://handlebarsjs.com/partials.html
+            } else if (node.type === handlebarsUtils.PARTIAL_EXPRESSION && 
+                parser.getCurrentState() !== stateMachine.State.STATE_DATA && 
+                (re = handlebarsUtils.isValidExpression(node.content, 0, node.type)) && 
+                (partialName = re.tag)) {
+
+                key = this._config._namespace === ''? partialName : this._config._namespace + '/' + partialName;
```

Comment:
this `namespace` serves as a prefix, where it is a suffix after partialName at https://github.com/yahoo/secure-handlebars/pull/120/files#diff-a621da0e5af95d7c9190225fa70efa85R46

pls correct it, and add a test case too. thanks.